doc/classes/XRFaceTracker.xml

```diff
@@ -0,0 +1,469 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="XRFaceTracker" inherits="RefCounted" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A tracked face.
+	</brief_description>
+	<description>
+		An instance of this object represents a tracked face and its corresponding blend shapes. The blend shapes come from the [url=https://docs.vrcft.io/docs/tutorial-avatars/tutorial-avatars-extras/unified-blendshapes]Unified Expressions[/url] standard.
+		As face trackers are turned on they are registered with the [XRServer].
+	</description>
+	<tutorials>
+		<link title="XR documentation index">$DOCS_URL/tutorials/xr/index.html</link>
+	</tutorials>
+	<methods>
+		<method name="get_blend_shape" qualifiers="const">
+			<return type="float" />
+			<param index="0" name="blend_shape" type="int" enum="XRFaceTracker.BlendShapeEntry" />
+			<description>
+				Returns the requested face blend shape weight.
+			</description>
+		</method>
+		<method name="set_blend_shape">
+			<return type="void" />
+			<param index="0" name="blend_shape" type="int" enum="XRFaceTracker.BlendShapeEntry" />
+			<param index="1" name="weight" type="float" />
+			<description>
+				Sets a face blend shape weight.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="blend_shapes" type="PackedFloat32Array" setter="set_blend_shapes" getter="get_blend_shapes" default="PackedFloat32Array(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0)">
```

Comment:
Insane. There are some places in the code base that use the **ADD_PROPERTY_DEFAULT** macro to override how the docs generate this value. I would heavily recommend using it here (and writing a comment as to why it is being done).

Reply:
Nice catch. I wasn't aware of ADD_PROPERTY_DEFAULT. This is now done.